keyboards/okuura/goaisatsu/readme.md

```diff
@@ -0,0 +1,17 @@
+# goaisatsu
+
+![goaisatsu](https://s2.booth.pm/7d7df4a1-8345-407f-9c2f-8eefe1f7a17f/i/4980888/1d7d5d1b-0eda-4db5-af38-a39bece6c12e_base_resized.jpg)
```

Comment:
could this image please be re-hosted on imgur, also including the `h` before .jpeg/.png in url  
e.g. `https://i.imgur.com/SK0flNs.jpeg` -> `https://i.imgur.com/SK0flNsh.jpeg`

Reply:
Thank you for all your suggestions😀 You have made this PR better.
link has also been changed to imgur.
https://i.imgur.com/rd0MFFlh.jpg